inc/compatibilities/sassy_social_share.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Class Optml_sassy_social_share.
+ *
+ * @reason Sassy social share picks eco images to share
+ */
+class Optml_sassy_social_share extends Optml_compatibility {
+	/**
+	 * Should we load the integration logic.
+	 *
+	 * @return bool Should we load.
+	 */
+	function should_load() {
+		include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+
+		if ( is_plugin_active( 'sassy-social-share/sassy-social-share.php' ) ) {
```

Comment:
let's return early here 
```suggestion
		if ( ! is_plugin_active( 'sassy-social-share/sassy-social-share.php' ) ) {
		    return false;
		}
```

Reply:
ok